storm/content.md

```diff
@@ -0,0 +1,59 @@
+# What is Apache Storm?
+
+Apache Storm is a distributed computation framework written predominantly in the Clojure programming language. Originally created by Nathan Marz and team at BackType, the project was open sourced after being acquired by Twitter. It uses custom created "spouts" and "bolts" to define information sources and manipulations to allow batch, distributed processing of streaming data. The initial release was on 17 September 2011.
+
+> [wikipedia.org/wiki/Storm_(event_processor)](https://en.wikipedia.org/wiki/Storm_(event_processor))
+
+%%LOGO%%
+
+# How to use this image
+
+This image entry point is a [Storm command line client](http://storm.apache.org/releases/1.0.0/Command-line-client.html). See couple of usage examples below.
+
+## Running topologies in local mode
+
+Assuming you have `storm-starter-topologies-1.0.1.jar` in the current directory.
+
+	$ docker run -it -v $(pwd)/storm-starter-topologies-1.0.1.jar:/topology.jar 31z4/storm:1.0.1 storm jar /topology.jar org.apache.storm.starter.ExclamationTopology
+
+## Setting up a minimal Storm cluster
+
+1.	[Apache Zookeeper](https://zookeeper.apache.org/) is a must for running a Storm cluster. Start it first. Since the Zookeeper "fails fast" it's better to always restart it.
+
+		$ docker run -d --restart always --name zookeeper jplock/zookeeper:3.4.8
+
+2.	The Nimbus daemon has to be connected with the Zookeeper. It's also a "fail fast" system.
+
+		$ docker run -d --restart always --name nimbus --net container:zookeeper 31z4/storm:1.0.1 storm nimbus
```

Comment:
Oh, they do exactly that down below in the `docker-compose` example -- perhaps this one could use `--link` instead?